functions/memorystore/redis/test/index.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2019, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const proxyquire = require('proxyquire').noCallThru();
+const sinon = require('sinon');
+const assert = require('assert');
+
+function getSample({error=false}) {
```

Comment:
Can we have these connect to an actual Redis instance (perhaps using the `nodejs-docs-samples-tests` GCP project)? [I added you as an Editor - but ping me if you need more access.]

@fhinkel and I are trying to phase out unit-only tests where possible.

Reply:
I'm not sure how to achieve that - the function under test would have to be deployed on GCF to connect to an actual Memorystore instance. Would we have the tests deploy and then delete the function? :thinking: